src/ZendeskApi_v2/Requests/Tickets.cs

```diff
@@ -74,6 +74,10 @@ public interface ITickets : ICore
 
         GroupTicketResponse GetMultipleTickets(IEnumerable<long> ids, TicketSideLoadOptionsEnum sideLoadOptions = TicketSideLoadOptionsEnum.None);
 
+        GroupTicketResponse GetMultipleTickets(IEnumerable<long> ids, int pageNumber, int itemsPerPage, TicketSideLoadOptionsEnum sideLoadOptions = TicketSideLoadOptionsEnum.None);
```

Comment:
Why are pageNumber and ItemsPerPage not nullable here but they are for the async version?

more to the point why do the methods differ more then the fact that they are async?
example: parameter names should be the same.